lib/jemoji.rb

```diff
@@ -4,7 +4,8 @@
 
 module Jekyll
   class Emoji
-    GITHUB_DOT_COM_ASSET_ROOT = "https://assets.github.com/images/icons/".freeze
+    GITHUB_DOT_COM_ASSET_HOST_URL = "https://assets.github.com".freeze
```

Comment:
I'll update this to `https://assets-cdn.github.com` as part of this pull request too.